### DISCUSSION TITLE
CROFANTOKEN

### DISCUSSION THREAD

**Discussion by Cryptalien** [crypto-org-chain/chain-main#753] [Announcements]
**Contact***
crofantoken@gmail.com @CRYPTALIEN5 telegram 
_**Team/Company Name (if any)**_
CROFANTOKEN
**Team/Company Website (if any)**
https://crofantoken.ga/
https://t.me/CroFanToken
**Project Description***
Token LP locker, staking platform, marketplace, launchpad, lottery, DAO
**Project Category***
Utility
**How will the project benefit the Crypto.org Chain and its ecosystem?***
This toke will bring the fans from the sports arenas and sponsorships that crypto.com is marketing to. The launchpad will give developers a place to launch their project and raise funds. The investors will have the safety and security with our project due to the audits and safety measures we took into consideration.
What is the timeline for implementing the project?*
We have completed the LP Locker. We are 22 days into the project. The rest of the ecosystem will be built in accordance with the speed at which we complete milestones. 
**Budget Plan***
We have spent money on development of the ecosystem. Now we are looking to incorporate, audit, develop the other parts of the ecosystem and most importantly use the budget for marketing and booking AMA’s for to bring awareness to the project.
**Project Status***
Locker is live and functional. There is multiple locks already locked and unlocked successfully. Next stage in the roadmap is staking, launchpad and marketplace.
Idea and Proposal
We want to bring a safe and economically viable lp locker option to the cronos network. The marketplace will use the Crown token to buy and sell products and services in NFT’s. The native token of the ecosystem will be the crowfantoken. The DAO will be a real estate DAO and the proceeds will go to the holders and contributors of the DAO. We will provide developer tools also on the website in order to make life easier on the developers.
**MVP**
**Beta**
https://crofantoken.ga/locker/#/locker
Official Launch
MARCH 30 2022
*Mandatory information.

**Comment by aw126** [comment]
Thanks for your interest in our grant program.
Your proposal has been received and we will forward the details to the relevant team who will work on your application.

**Comment by DegenApeDev** [comment]
Legit team looking to make utilities to fix everyday problems that people face in the cronos chain network. First utility was a locker and the team delivered on their promises. Excited to see what else this team brings to the table.

**Reply by Cryptalien** [reply]
The LP LOCKER is live. Crossing to other chains very soon. Next stage in development is going to be a Launchpad. The tools for developers and projects will be available also to increase the exposure of tokens launched on the CROFANPAD.
